src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -117,6 +118,12 @@ public ProjectsListViewController(final Model model, final Controller controller
             searchTextField.setText("");
          }
       });
+      searchTextField.addEventFilter(MouseEvent.MOUSE_CLICKED, e -> {
```

Comment:
can you try to listen "onfocus" instead? So that normal mouse clicks still move the cursor position after the first click

Reply:
Fixed it with an Listener on the Focused Property